WebDriverAgentLib/FBApplication.m

```diff
@@ -21,36 +23,70 @@
 #import "XCUIElementQuery.h"
 #import "FBXCAXClientProxy.h"
 #import "XCUIApplicationProcessQuiescence.h"
+#import "XCTestManager_ManagerInterface-Protocol.h"
+#import "XCTestPrivateSymbols.h"
+#import "XCTRunnerDaemonSession.h"
+
+static const NSTimeInterval APP_STATE_STABILITY_WINDOW = 1.0;
+static const NSTimeInterval APP_STATE_STABILITY_TIMEOUT = 5.0;
 
 @interface FBApplication ()
 @property (nonatomic, assign) BOOL fb_isObservingAppImplCurrentProcess;
 @end
 
 @implementation FBApplication
 
++ (nullable XCAccessibilityElement *)fb_currentAppElement
+{
+  CGPoint screenPoint = CGPointMake(100, 100);
```

Comment:
how does it work with iPad and split screen ?

Reply:
(I'm a bit sceptical about trying to get active app by hardcoded coords.)

Reply:
Yeah, hardcoded values are not ideal, we could get the size of the system application and then tap by percentage. Although, the problem of active application determining stays open. My theory is that "active" is the app, which is currently at the left top corner of the screen (the place where a human would start to "scan" the screen).

Reply:
now the coordinates are not hardcoded anymore - only the relative offset is hardcoded